apps/docs/src/guide/testing/index.md

```diff
@@ -8,36 +8,47 @@
 
 In order to test your Sway and TS-SDK applications, you can test your code in a number of ways:
 
-1. Testing with TS-SDK: Compiling you Sway code and connecting to the methods using TS-SDK and JS testing frameworks
-2. Using `forc test` see <a :href="url" target="_blank" rel="noreferrer">the Sway docs</a> for more info
-3. Using [the Rust SDK](https://docs.fuel.network/docs/fuels-rs/testing/)
+1. Testing with the TS-SDK: Compiling you Sway code and connecting to the methods using TS-SDK and JS testing frameworks
+1. Using [`forc test`](https://docs.fuel.network/docs/forc/commands/forc%5ftest/#forc-test)
+1. Using [the Rust SDK](https://docs.fuel.network/docs/fuels-rs/testing/)
 
-### Testing with TS-SDK
+## Testing with the TS-SDK
 
 To test your Sway applications using the TS-SDK, you can pick whatever testing library or framework you feel comfortable with. There isn't any specific testing framework needed, it is entirely up to the user. That being said, the TS-SDK uses [`Vitest`](https://vitest.dev/) for its tests.
 
-### Wallet Test Utilities
+### Launching a Test Node
 
-You'll often want to create one or more test wallets when testing your contracts.
+To simplify testing in isolation, the SDK provides `launchTestNode`, a utility via which you can spin up a short-lived `fuel-core` node, setup a custom provider and wallets, and deploy contracts in one go.
+Here is a simple contract deployment in a test:
 
-For this, you can find two simple utilities on the account package:
+<<< @/../../docs-snippets/src/guide/testing/launching-a-test-node.test.ts#deploy-contract{ts:line-numbers}
 
-- [`@fuel-ts/account`](https://github.com/FuelLabs/fuels-ts/tree/master/packages/account#test-utilities)
+- The `launched` variable was instantiated with the [`using`](https://devblogs.microsoft.com/typescript/announcing-typescript-5-2/#using-declarations-and-explicit-resource-management) declaration. This is a TypeScript 5.2 feature called explicit resource management and it's used to automatically kill the node once `launched` goes out of scope.
```

Comment:
The launched explanation could precede the snippet, followed by the `tsconfig.json` config example and then the snippet.